tests/validation/tests/v3_api/test_cis_scan.py

```diff
@@ -25,6 +25,9 @@
 from .test_rke_cluster_provisioning import HOST_NAME, \
     POD_SECURITY_POLICY_TEMPLATE, get_cis_rke_config  # NOQA
 
+# Skipping all the tests from this module since CIS 1.5 and CIS 1.4 profile are not supported on latest k8s versions.
+pytest.skip(allow_module_level=True)
```

Comment:
@sowmyav27 should only the 1.4 scans be disabled here? or both.

Reply:
Both technically.

Reply:
Since 2.6 supports k8s 1.21 and below shouldn't we have cis 1.5?

Reply:
As per this issue https://github.com/rancher/rancher/issues/38668 CIS 1.5 profile is hidden now, as it supports k8s 1.15.